src/RageFileManager.h

```diff
@@ -74,12 +77,18 @@ class RageFileManager
 
 	bool Unzip(const std::string &zipPath, std::string targetPath, int strip);
 
+	// path protection
+	void ProtectPath(const std::string& path);
+	bool IsPathProtected(const std::string& path);
+
 	// Lua
 	void PushSelf( lua_State *L );
 
 private:
 	RageFileBasic *OpenForReading( const RString &sPath, int iMode, int &iError );
 	RageFileBasic *OpenForWriting( const RString &sPath, int iMode, int &iError );
+
+	std::unordered_set<std::string> protectedPaths;
```

Comment:
I think for the sake of consistency with the rest of the code base, we should call this `m_protectedPaths`

Reply:
done